apps/backend/config-server/src/main/resources/application.yaml

```
@@ -0,0 +1,21 @@
+          search-locations:
+            - classpath:/config/file-service
+            - classpath:/config/gateway
+            - classpath:/config/sync-service
+            - classpath:/config/user-service
+eureka:
+  client:
+    service-url:
+      defaultZone: ${EUREKA_CLIENT_SERVICEURL_DEFAULTZONE:http://localhost:8761/eureka/}
+      register-with-eureka: true
+server:
+  port: 8071
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)